Bugfix/acl create

EntonoX opened the pull request:
Fix issue where ACL plugin 1.0-x is using different create method for ACL's.

Douglasdc3 commented:
Checked this against the docs. The consumer is only needed in the url definition not in the body of the call.

I think it would be better and still work if the consumer_id property was removed from the ACL Consumer completely and of course references to it.

EntonoX commented:
> Checked this against the docs. The consumer is only needed in the url definition not in the body of the call.
> 
> I think it would be better and still work if the consumer_id property was removed from the ACL Consumer completely and of course references to it.

You were totally right. I removed the unneeded parameter from the create function. Also changed to consumer variable in AclConsumer model. Tested with several, list, create and manual acl create. 

I added these fixes to the branch.
